spec/dummy/client/app/startup/clientGlobals.jsx

```diff
@@ -6,9 +8,16 @@ import HelloWorldApp from './HelloWorldApp';
 import ClientRouterApp from './ClientRouterApp';
 
 // This is an example of how to render a React component directly, without using Redux
-window.HelloWorld = HelloWorld;
+// window.HelloWorld = HelloWorld;
 window.HelloWorldWithLogAndThrow = HelloWorldWithLogAndThrow;
 window.HelloWorldES5 = HelloWorldES5;
 window.ReduxApp = ReduxApp;
 window.HelloWorldApp = HelloWorldApp;
 window.RouterApp = ClientRouterApp;
+
+ReactOnRails.registerComponent('HelloWorld', HelloWorld);
```

Comment:
It's not working as expected. Debugged from this call, it shows we defined `registerComponent ` incorrectly.